suite-common/wallet-core/src/transactions/transactionsReducer.ts

```diff
@@ -148,3 +150,15 @@ export const selectAccountTransactions = (
     state: TransactionsRootState,
     accountKey: string | null,
 ) => state.wallet.transactions.transactions[accountKey ?? ''] ?? [];
+
+export const selectAllTransactions = createSelector(
+    selectTransactions,
+    (transactions): WalletAccountTransaction[] => Object.values(transactions).flat(),
+);
+
+export const selectTransaction = (txid: string) =>
```

Comment:
```suggestion
export const selectTransactionById = (txid: string) =>
```
What if we started with suffixing these specific parameterized selectors according to the parameter/parameters by which it is selected?

Reply:
I kind of like but I am also quite worried that it will complicate current convention. Currently variable name is always same as selector without prefix aka `const transaction = selectTransaction(state, txid)` which is super simple to follow. It should probably work also with also removing `BySomething` suffix but it starting to be little more complicated.

Overall it's improve readability quite a lot so let's give it a try. +1 from me

Reply:
Also I think it should be `selectTransactionByTxid`

Reply:
Yeah I agree. It will make it more explicit.